test/test_flake8.py

```diff
@@ -11,13 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from flake8.api.legacy import get_style_guide
 
-from ament_flake8.main import main
 import pytest
 
 
 @pytest.mark.flake8
 @pytest.mark.linter
 def test_flake8():
-    rc = main(argv=[])
-    assert rc == 0, 'Found code style errors / warnings'
+    style = get_style_guide(
+        ignore='I202',
+        import_order_style='google',
```

Comment:
As @mikaelarguedas mentioned I don't think this option is currently being applied.

Reply:
This seems to be the case. Without flake8-import-order installed flake8 logs an error when this option is used. With flake8-import-order installed it does not, but doesn't seem to report errors with import order when invoked via test_flake8.py.

Should we switch to a configuration file instead?

Reply:
Well, currently the code fails the test (at least for me). And changing the argument for the import order style in the test doesn't make any difference.

Only with a `.flake8` file with the following content it does pass for me:

```
[flake8]
import-order-style = google
```